### ISSUE TITLE
Use the GitHub's statuses API

### ISSUE THREAD

**Issue by skateman** [ManageIQ/miq_bot#91]
GitHub just announced a new [status bar](https://github.com/blog/2040-clearer-mergability-information-for-pull-requests) under the comment section of the PRs. I think that the lint reviews would look better by using the [statuses API](https://developer.github.com/v3/repos/statuses/).

**Comment by Fryguy** [comment]
Agreed...we'd have to publicly host the results somewhere though...still...might be pretty sweet

Either that or just link to the comment for now.

**Comment by chriskacerguis** [comment]
@miq-bot set_milestone Bot v2
